enterprise/signals.py

```diff
@@ -97,6 +99,20 @@ def _complete_user_enrollment():
         pending_ecu.delete()
 
 
+@receiver(post_save, sender=EnterpriseCustomerBrandingConfiguration)
```

Comment:
I think we are saving the same file with different names. One in this function and one at [here](https://github.com/edx/edx-enterprise/blob/599cc21df6910afc6c2ed82c202a2ad80ab1f778/enterprise/models.py#L1129).

Please take a look at this either this cover this ticket scope or not. Other than that LGTM and Good Catch 👍